src/server/pps/server/s3g_sidecar_test.go

```diff
@@ -772,3 +822,22 @@ func TestS3SkippedDatums(t *testing.T) {
 		}
 	})
 }
+
+func TestS3SkippedDatums(t *testing.T) {
+	if testing.Short() {
+		t.Skip("Skipping integration tests in short mode")
+	}
+	c, userToken, ns := initPachClient(t)
+	if userToken == "" {
+		userToken = "abc123"
+	}
+	t.Run("ProjectUnaware", func(t *testing.T) {
+		testS3SkippedDatums(t, c, userToken, userToken, ns, pfs.DefaultProjectName, func(p, r string) string { return r })
+	})
+	t.Run("ProjectAware", func(t *testing.T) {
+		accessKeyID := "PAC1" + userToken
+		projectName := tu.UniqueString("project")
+		require.NoError(t, c.CreateProject(projectName))
+		testS3SkippedDatums(t, c, accessKeyID, userToken, ns, projectName, func(p, r string) string { return r + "." + p })
```

Comment:
Is this convention of `r + "." + p` just used for the sake of testing here and is isolated from the discussion with Matthew regarding how to name buckets with projects moving forward?

Reply:
It’s due to this particular test using _both_ the global _and_ the sidecar S3 gateways; the global gateway bucket name schema is different for project-aware clients. The bucket names when used as a sidecar do not change.